src/PayID/pay-id-client.ts

```diff
@@ -197,10 +195,10 @@ export default class PayIDClient {
 
     const travelRule = new TravelRule(originator, beneficiary)
 
-    const compliance = new Compliance(complianceType, travelRule)
+    const compliance = new Compliance(ComplianceType.TravelRule, travelRule)
 
     const signatureWrapper = new SignatureWrapperCompliance(
-      complianceType,
+      'compliance',
```

Comment:
These strings are uppercase in the WP

Not sure if that really affects functionality, but I would say it would be best to keep those the same

Reply:
To be clear, not all uppercase, but like: `Compliance`